File: Backend/ElBuenSabor/src/main/java/com/utnfrm/entities/DetalleFactura.java
package com.utnfrm.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "detallesFacturas")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
//@Audited
public class DetalleFactura extends Base {

    @Column
    private int cantidad;

    @Column
    private Double subtotal;

    @ManyToOne
    @JoinColumn(name = "fk_articulo_manufacturado")
    private ArticuloManufacturado articuloManufacturado;

    @ManyToOne
    @JoinColumn(name = "fk_articulo_insumo")
    private ArticuloInsumo articuloInsumo;
}

File: Frontend/el-buen-sabor/src/app/models/detalle-factura.ts
import { ArticuloInsumo } from './articulo-insumo';
import { ArticuloManufacturado } from './articulo-manufacturado';
import { Generic } from './generic';

export class DetalleFactura extends Generic {
  cantidad: number;
  subtotal: number;
  articuloManufacturado: ArticuloManufacturado;
  articuloInsumo: ArticuloInsumo;
}
